OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -27,6 +26,9 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <Optimize>false</Optimize>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)' == 'Release-x86'">
```

Comment:
Not sure how important, but the spacing is different from the Red Alert entry below.

Reply:
Added a new commit to fix the whitespace in the other entries.